src/contracts/multichain/CrossChainRegistry.sol

```diff
@@ -338,15 +305,6 @@ contract CrossChainRegistry is
 
             emit TransportDestinationRemoved(operatorSet, chainID);
         }
-
-        // Check final state based on isDelete flag
-        if (!isDelete) {
-            // For normal removal, at least one destination should remain
-            require(_transportDestinations[operatorSetKey].length() > 0, RequireAtLeastOneTransportDestination());
```

Comment:
This check needs to exist while deleting the transport destinations right? i.e we need atleast one chain.